Infrastructure/FileLogger.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace Infrastructure
+{
+    public class FileLogger : ILogger
+    {
+        private readonly object fileLock = new object();
+        private readonly string datetimeFormat;
+        private string logFilename;
+        string pathPrefix = "../../";
```

Comment:
Please use pathPrefix in the constructor while creating a file but not only when appending